features/dd-sdk-android-rum/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScopeTest.kt

```diff
@@ -903,77 +907,6 @@ internal class RumViewScopeTest {
         assertThat(result2).isNull()
     }
 
-    @Test
-    fun `𝕄 send event 𝕎 handleEvent(StopView) on active view`(
```

Comment:
why do we remove this test? I can see similar test is added with synthetics session, but then shouldn't this test assert user session type then instead of removing it?

Reply:
That must have been a cut-paste instead of a copy paste, fixed